cfme/tests/control/test_basic.py

```diff
@@ -256,13 +266,18 @@ def alert_profile_collection(appliance):
 
 
 @pytest.yield_fixture
-def two_random_policies(policy_collection):
+def two_random_policies(appliance, policy_collection):
+    # Physical Infrastucture policies excluded
+    if appliance.version < "5.9":
+        policies = COMPLIANCE_POLICIES[:-1] + CONTROL_POLICIES[:-1]
```

Comment:
**Required**

Make this remove the correct policy dynamically. If another policy is added to either COMPLIANCE_POLICIES or CONTROL_POLICIES this will stop working correctly.

Reply:
@mshriver I'm pretty sure no new policies will be add :)